Sources/Core/StringExtensions.swift

```diff
@@ -52,31 +52,114 @@ prefix func --> (body: () -> [String]) -> String {
     return -->body()
 }
 
+// Most of these are derived from https://www.binpress.com/tutorial/objective-c-reserved-keywords/43
```

Comment:
We should create some abstraction for each language. Could be for example a `Language` protocol that will be passes around and every supported language  creates a struct that conforms to it.

Reply:
@maicki - I agree since there will be different reserved words per language. Only caveat is that the names must be consistent for any languages that bridge (Objc, Flow, Swift, etc.) so we might need a global set of all languages at runtime to utilize for consistent results.